src/components/core/tabs/tab.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
+import css from './tab.module.scss';
+
+export interface TabProps {
+    active?: boolean;
+    active_tab_ref?: any;
+    contained?: boolean;
+    dark?: boolean;
+    icon?: string;
+    label?: string;
+    onClick?: () => void;
+}
```

Comment:
```suggestion
export interface TabProps extends LiHTMLAttributes<HTMLDataListElement> {
    active?: boolean;
    contained?: boolean;
    dark?: boolean;
    icon?: string;
    label?: string;
}
```

Lets used the built in type definition and extend it to include our own custom types